LoggingTests/LoggingTests.swift

```
@@ -11,45 +11,42 @@ import XCTest
-    func testFileFound() {
-       
-        let path = FileManager.default.urls(for: .documentDirectory, in: .userDomainMask).last
-        let url = path?.appendingPathComponent("Logging.json", isDirectory: false)
-        var result = Helper.checkForFile(filePath: url!)
-        if !result {
-            Form781Controller.shared.create(date: Helper.getTodaysDate(), mds: "C017A", serialNumber: "90-0534", unitCharged: "437 AW (AMC) /DKFX", harmLocation: "JB CHARLESTON SC 29404", flightAuthNum: "20-0772", issuingUnit: "0016AS")
-            result = Helper.checkForFile(filePath: url!)
-        }
-        XCTAssertTrue(result)
-    }
+    //This creates a file if the file isn't there and then returns true.
+    //This is testing file creation, not file not found
+//    func testFileFound() {
+//
+//        let path = FileManager.default.urls(for: .documentDirectory, in: .userDomainMask).last
+//        let url = path?.appendingPathComponent("Logging.json", isDirectory: false)
+//        var result = Helper.doesFileExist(atURL: url!)
+//        if !result {
+//            Form781Controller.shared.create(date: Helper.getTodaysDate(), mds: "C017A", serialNumber: "90-0534", unitCharged: "437 AW (AMC) /DKFX", harmLocation: "JB CHARLESTON SC 29404", flightAuthNum: "20-0772", issuingUnit: "0016AS")
+//            result = Helper.doesFileExist(atURL: url!)
+//        }
+//        XCTAssertTrue(result)
+//    }
     
-    func testCheckInput() {
-        let result = Helper().checkInput(time: "0900")
-        XCTAssertTrue(result)
-    }
     
-    func testSeperateHours() {
-        // Test requires input to have a colon
-        let hrs = Helper.separateHoursAndMins(strInput: "10:00", pointer: "hour")
-        XCTAssertEqual(hrs, "10")
+    func testStringExtensions() {
+        XCTAssertTrue("0900".isExactlyFourCharacters())
+        XCTAssertFalse("090".isExactlyFourCharacters())
+        
+        XCTAssertTrue("0123".isDigits)
+        XCTAssertFalse("123A".isDigits)
```

So we are not testing the "00:00" format anymore? This test was supposedly written for a reason. Please, please, please, DO NOT remove unit tests unless the test is agreed to be invalid.

---

The seperateHoursAndMins function was removed, so no longer needs a test. 
The function using that now just removes the colon and continues with a 4 digit string. 

I'm not removing anything until is agreed. This is a reviewable pull request for this purpose.

---

Yes, I'm a little concerned that we are not as thorough handling of ##:## anymore, but I guess if things start to silently fail, we will figure it out.

---

It's being handled by removing the ":" from the string. The UI is also handling this as it doens't allow it.